### PR TITLE
OI-305 Change footer logo path. Add script that changes background ur…

### PR DESCRIPTION
**Pull request by nvelychenko**
…l for openideal toolbar logo icon.

### CODE REVIEW

File: modules/openideal_toolbar/js/toolbar.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+     * Openideal Toolbar behavior.
+     *
+     * @type {Drupal~behavior}
+     *
+     * @prop {Drupal~behaviorAttach} attach
+     *   Attach icon for OI toolbar.
+     */
+    Drupal.behaviors.openidealToolbar = {
+      attach: function (context, settings) {
+        $('.toolbar-icon-openideal-toolbar-help').once('openideal_toolbar').each(function () {
```
Comment (db-Matroskeen) [inline modules/openideal_toolbar/js/toolbar.js]: Should we add `context`?

File: modules/openideal_toolbar/js/toolbar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          var logo = settings.openidealToolbar.logo;
+          $(this).css('--toolbar-url', `url(${logo})`)
```
Comment (db-Matroskeen) [inline modules/openideal_toolbar/js/toolbar.js]: Should we check if the logo is not empty? (in case someone removed it in any hook_alter)

**Review by db-Matroskeen — CHANGES_REQUESTED** [review]

File: modules/openideal_toolbar/js/toolbar.js (hunk 1 above)
Comment (nvelychenko) [reply modules/openideal_toolbar/js/toolbar.js]: Done

File: modules/openideal_toolbar/js/toolbar.js (hunk 2 above)
Comment (nvelychenko) [reply modules/openideal_toolbar/js/toolbar.js]: Added additional condition

**Pull request merged**